.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "geoblacklight"]
 	path = geoblacklight
 	url = https://github.com/CottageLabs/geoblacklight.git
+
+[submodule "willow_sword"]
+	path = willow_sword
+	url = git@github.com:CottageLabs/willow_sword.git
```

Comment:
A server (like the demo server) will have a hard time doing `git submodule update` on a submodule that requires an SSH key to clone. Usually https (see above) is used for submodules.